.github/workflows/test_nrf9160dk.yml

```diff
@@ -94,15 +94,21 @@ jobs:
     needs: build_for_hw_test
     runs-on: [self-hosted, has_nrf9160dk]
 
+    defaults:
+      run:
+        working-directory: connection_tests
+
     steps:
       - name: Checkout repo
         uses: actions/checkout@v3
+        with:
+          path: connection_tests
 
       - name: Download build tarball
         uses: actions/download-artifact@v3
         with:
           name: build.tar.gz
-          path: samples/test
+          path: connection_tests/samples/test
```

Comment:
Change is fine, but referencing commit by sha1 won't work after this PR will be merged using GitHub, as GitHub does automatic rebase and sha1 of the referenced commit will change. So maybe just drop this sha1 and leave commit title only.